addons/mail/static/src/models/thread_needaction_preview_view.js

```diff
@@ -49,6 +71,18 @@ registerModel({
         },
     },
     fields: {
+        inlineLastNeedactionMessageAsOriginThreadBody: attr({
+            compute: '_computeInlineLastNeedactionMessageAsOriginThreadBody',
+            readonly: true,
+        }),
+        /**
+         * States if the threadNeedactionPreview has no messages or has a message with empty body.
+         */
+        isEmpty: attr({
+            compute: '_computeIsEmpty',
+            default: false,
```

Comment:
In regards to comment from @alexkuhn I really find this is making the code very confusing to rely on default for this kind of compute.

Reply:
The `clear` is correct, and reliance on default makes totally sense with
```js
if (!this.lastNeedactionMessageAsOriginThread) {
```

But for some reasons, the code was wrongfully changed with
```js
if (this.inlineLastNeedactionMessageAsOriginThreadBody) {
```

`clear` is preferred because `isEmpty` makes sense when there's a `lastNeedactionMesageAsOriginThread`. But when there's no such message, it's basically `N/A`, so that's what the `clear` is for.